setup.cfg

```diff
@@ -78,7 +78,7 @@ tests =
     pytest-cov>=2.5.1
     uvtools
 vis =
-    fftvis>=0.0.7
+    fftvis@git+https://github.com/tyler-a-cox/fftvis@mp_fftvis
     line-profiler
     matvis>=1.2.1
```

Comment:
I think the only thing we need to do is update the matvis version to >=1.3.0. Other than that, this all looks fantastic, thank you!

Reply:
Good catch, updated.